pybind_interface/pybind_main.cpp

```diff
@@ -354,10 +363,8 @@ py::array_t<float> qsim_simulate_fullstate(const py::dict &options) {
 // TODO(95-martin-orion): add support for measurement gate conversion.
 std::vector<unsigned> qsim_sample(const py::dict &options) {
```

Comment:
It seems that measurements are not reproducible in Cirq. To emulate this behavior, a random seed should be set below.

Reply:
Added seed pass-through behavior for both this case and the full state-vector case.

Since the random value generation for Cirq and qsim are in Python and C++ respectively, I don't think there's a clean way to get identical results from each even if the same seed is used. However, qsim results should be reproducible in qsim now.

Reply:
Sure, there is no way to get identical results. I just meant that it seems that Cirq gives different results from invocation to invocation. qsim results depend on the provided seed and previously the results were always the same because the default seed was used.